qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -143,3 +146,4 @@
             gagent_serial_type = isa
             serials += " org.qemu.guest_agent.0"
             gagent_start_cmd = "pgrep qemu-ga || qemu-ga -d -m isa-serial -p /dev/ttyS1"
+            gagent_status_cmd = "pgrep qemu-ga"
```

Comment:
I saw the test steps in polarion is "#systemctl status qemu-guest-agent" after init 3. The status should be "Active: active (running)". What I concern is, will "pgrep qemu-ga" have the same effect with "systemctl"? And I saw that in other sub cases, pgrep is also used to check running status..

Reply:
For linux guest, we can check service status with "#systemctl status qemu-guest-agent". But for windows guest, it is just a background process.

Reply:
Didn't you disabled windows test in line 143? So why we should consider the situation of windows process?

Reply:
I have updated previous comment, sorry for mislead you.

Reply:
Alright. That's fine. I just wanna confirm with you that the way you used to check ga running is OK.

Acked-by: Xiyue Wang xiywang@redhat.com